### ISSUE TITLE
Only supports .NETCoreApp, rather than .NETStandard

### ISSUE THREAD

**Issue by snickler** [ststeiger/PdfSharpCore#19]
The first line of the description states:

`PdfSharpCore is a partial port of PdfSharp.Xamarin for .NET Standard `

This is misleading, as it simply targets .NETCoreApp, which means I won't be able to include this in a .NETStandard library.

![image](https://user-images.githubusercontent.com/4016293/45269185-97fe9b00-b457-11e8-9e6c-8163b9c8672e.png)

**Comment by ststeiger** [comment]
I've converted the 1.5 PdfSharp to NetStandard. 
Here: https://github.com/ststeiger/PdfSharpNetStandard

**Comment by haf** [comment]
@ststeiger Surely you can target this repo to net standard, as that can be used from .net core app x.x?

**Comment by haf** [comment]
Should I use this code or the other repo's?

**Comment by ststeiger** [comment]
@haf: Depends. 
If you need NetStandard, then the other repo ;) 
Fact of the matter is, Azure shared hosting doesn't allow calls to gdi32.dll (memory leaks). 
So, the NetStandard repository uses System.Graphics, and that calls gdi32.dll. 
On the other hand, the NetStandard-repository is a complete port of the latest version of PdfSharp and MigraDoc. 
This repo has been mutilated by missing apis in NetCore 1.x, but on the upside, it doesn't use System.Graphcis, so it could work on Azure shared hosting. 

In other words, if you need Azure shared hosting, use PdfSharpCore - otherwise you're much better of with PdfSharpNetStandard.
